client/app/components/group/group.controller.js

```diff
@@ -1,10 +1,18 @@
 class GroupController {
-  constructor($state, $mdMedia) {
+  constructor($state, $scope, $mdMedia) {
     "ngInject";
 
     if ($state.current.name === "group"){
       $state.go("group.groupDetail.pickups");
     }
+    
+    $scope.$watch(() => {
```

Comment:
Sadly I couldn't think of a better way to make the group breadcrumbs work. Happy if one of you guys could think of something without $scope.watch